Library/Homebrew/extend/git_path.rb

```diff
@@ -101,14 +101,19 @@ def git_commit_message(commit = "HEAD", safe: false)
     popen_git("log", "-1", "--pretty=%B", commit, "--", safe: safe, err: :out)&.strip
   end
 
+  sig { returns(Pathname) }
+  def pathname
```

Comment:
this is better expressed as `alias pathname __getobj__`, but then we have to add something like [this](https://github.com/Homebrew/brew/blob/master/Library/Homebrew/options.rbi)